src/types/User.ts

```diff
@@ -1,6 +1,7 @@
 type User = {
   email: string
-  confirmed: boolean
+  isConfirmed: boolean
+  confirmationSentAt?: string
```

Comment:
This adjustment is for API changes found in this PR: https://github.com/ProjectProtocol/project-protocol-api/pull/104